.github/workflows/repo-checks-tests.yml

```diff
@@ -45,6 +45,10 @@ jobs:
       E2E_TEST_AEST_MOF_OPERATIONS_USER: ministry-user-mof-operations@e2e-tests
       E2E_TEST_AEST_READ_ONLY_USER: ministry-user-read-only@e2e-tests
       E2E_TEST_PASSWORD: ${{ secrets.E2E_TEST_PASSWORD }}
+      MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT: ${{ secrets.MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT }}
+      MAXIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER: ${{ secrets.MAXIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER }}
+      MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION: ${{ secrets.MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION }}
+      MAXIMUM_IDLE_TIME_FOR_WARNING_AEST: ${{ secrets.MAXIMUM_IDLE_TIME_FOR_WARNING_AEST }}
```

Comment:
Since this is not sensitive data, it can have the values set in the `make` file and we would not need them added here.

Reply:
No need to keep this changes unless I am missing something. Please let me know.